helm-charts/sep-service/templates/configmap.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.fullName }}-configmap-{{ .Values.service.name }}
+  namespace: {{ .Values.namespace }}
+data:
+  # TODO: remove all secrets (ex: move to env vars, k8s secrets, vault, etc...)
+  anchor-config.yaml: |
+    stellar:
+      anchor:
+        config: in-memory
+        app-config:
+          type: config-spring-property                # Activate [config-spring-property] module
+          settings: app-config                        # The location of the configuration data
+        data-access:
+          type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+          settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
+        logging:
+          type: logging-logback
+          settings: logging-logback-settings
+    app-config:
+      app:
+        stellarNetworkPassphrase: Test SDF Network ; September 2015
```

Comment:
Should we template this? We could have TESTNET and PUBNET in the values file and derive passphrase from there.